packs/nest/actions/fan.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: fan
+runner_type: run-python
+description: Manage fan state
+enabled: true
+entry_point: fan.py
+parameters:
+  state:
+    type: string
+    description: State to set the fan
+    enum:
+      - "on"
+      - auto
+    default: auto
+    required: true
+  structure:
+    type: integer
+    description: ID of Nest Home name
+    default: 0
+  device:
```

Comment:
It might make sense to do what we do with some other actions - allow user to set global default for `device` and `structure` in the config, but also allow to override it on per action basis.